packages/core/src/index.ts

```diff
@@ -6,7 +6,9 @@ export type {
   MarginProps,
   PaddingProps,
   PickUnion,
+  Simplify,
 } from './types';
 export * from './Flex';
 export * from './Grid';
 export * from './Box';
+export { getSpacingStyles } from './utils/getSpacingStyles';
```

Comment:
I guess that we can use the Box component in Icon directly, so we don't need to export this utility function.